Library/Homebrew/cmd/tap.rb

```diff
@@ -28,14 +32,16 @@ def install_tap user, repo
     tapd.find_formula { |file| files << file }
     puts "Tapped #{files.length} formula#{plural(files.length, 'e')} (#{tapd.abv})"
 
-    if private_tap?(repouser, repo) then puts <<-EOS.undent
-      It looks like you tapped a private repository. To avoid entering your
-      credentials each time you update, you can use git HTTP credential caching
-      or issue the following command:
+    unless clone_target
```

Comment:
@xu-cheng I see what you mean. I would rather make that a tiny method then.

That is, I would prefer this:

    if check_private?(clone_target, repouser, repo)

to this:

    if not clone_target && private_tap?(repouser, repo)`

Any objections to that before I make the change?